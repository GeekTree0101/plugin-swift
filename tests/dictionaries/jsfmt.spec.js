run_spec(__dirname, ["swift"]);
run_spec(__dirname, ["swift"], { trailingComma: "all" });
