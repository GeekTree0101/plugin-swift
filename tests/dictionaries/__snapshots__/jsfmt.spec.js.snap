// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dictionaries.swift 1`] = `
let oneElement = ["answer": 42]
let twoElements = ["apple": 1, "banana": 2]
let someDictionaryExpressionThatShouldBreak = ["becauseItHasOneVeryLongElement": "thatForcesItToBreak"]
let someDictionaryExpressionThatShouldAlsoBreak = ["because": "itHas", "soLongElementsInIt": "thatItHasTo"]
let someDictionaryExpressionThatIsJustPrintedOfTheDocumentPrintWidthBoundary = [:]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
let oneElement = ["answer": 42]
let twoElements = ["apple": 1, "banana": 2]
let someDictionaryExpressionThatShouldBreak = [
    "becauseItHasOneVeryLongElement": "thatForcesItToBreak"
]
let someDictionaryExpressionThatShouldAlsoBreak = [
    "because": "itHas",
    "soLongElementsInIt": "thatItHasTo"
]
let someDictionaryExpressionThatIsJustPrintedOfTheDocumentPrintWidthBoundary = [:]

`;

exports[`dictionaries.swift 2`] = `
let oneElement = ["answer": 42]
let twoElements = ["apple": 1, "banana": 2]
let someDictionaryExpressionThatShouldBreak = ["becauseItHasOneVeryLongElement": "thatForcesItToBreak"]
let someDictionaryExpressionThatShouldAlsoBreak = ["because": "itHas", "soLongElementsInIt": "thatItHasTo"]
let someDictionaryExpressionThatIsJustPrintedOfTheDocumentPrintWidthBoundary = [:]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
let oneElement = ["answer": 42]
let twoElements = ["apple": 1, "banana": 2]
let someDictionaryExpressionThatShouldBreak = [
    "becauseItHasOneVeryLongElement": "thatForcesItToBreak",
]
let someDictionaryExpressionThatShouldAlsoBreak = [
    "because": "itHas",
    "soLongElementsInIt": "thatItHasTo",
]
let someDictionaryExpressionThatIsJustPrintedOfTheDocumentPrintWidthBoundary = [:]

`;
